tests/index.spec.ts

```diff
@@ -0,0 +1,35 @@
+import type { ExecaChildProcess } from "execa";
+import { execa } from "execa";
+import * as path from "path";
+import { fileURLToPath } from 'url';
+
+const EXTENSIONS = ".ts,.tsx,.cts,.mts";
+const NO_IGNORE_FLAG = "--no-ignore";
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+
+const eslintModulePath = path.resolve(__dirname, "../node_modules/.bin/eslint");
+
+async function lintProject(projectName: string): Promise<ExecaChildProcess<string>> {
+  const projectPath = path.resolve(__dirname, "fixtures", projectName);
+  const commandArguments = ["--ext", EXTENSIONS, NO_IGNORE_FLAG];
+
+  try {
```

Comment:
Why are we wrapping this in try/catch and not just let it throw the error if any?

Reply:
I had the same question/feeling and I already tried. Unfortunately, I couldn't manage to make it work, and I wasn't even able to print out the error thrown. So I've just followed the [Errors handling section](https://github.com/sindresorhus/execa#handling-errors) in the doc, and it shows an example where the error is caught (and we have a description of how the error looks)

Reply:
Ok thanks for the explanation!